cmd/servicelog/post.go

```diff
@@ -87,14 +89,37 @@ func (o *PostCmdOptions) Validate() error {
 	return nil
 }
 
+func (o *PostCmdOptions) CheckServiceLogsLastHour() bool {
+	getAllMessages := false      // we need just manual entries
+	getInternalLogsOnly := false // we need all messages
+	numberOfHours := 1           // number of hours we need to wait for svc logs
+	response, err := FetchServiceLogs(o.ClusterId, getAllMessages, getInternalLogsOnly)
+	if err != nil {
+		log.Fatalf("failed to fetch service logs: %q", err)
+	}
+	timeStampToCompare := time.Now().Add(-time.Hour * time.Duration(numberOfHours))
+	for _, svclog := range response.Items().Slice() {
+		timestamp := svclog.Timestamp()
+		if timestamp.After(timeStampToCompare.UTC()) {
+			fmt.Println("Below service Log has been subitted in last 60 minutes\nDescription: ", svclog.Description())
```

Comment:
I think it would be cleaner to print this line just once in case we had service logs in the past hour, then print the list of service logs - but I'll leave this up to you whether you want to change this or not.